codegenerator/cli/npm/envio/src/Logging.res

```diff
@@ -155,6 +155,7 @@ let getEventLogger = (eventItem: Internal.eventItem) => {
           "chainId": eventItem.chain->ChainMap.Chain.toChainId,
           "block": eventItem.blockNumber,
           "logIndex": eventItem.logIndex,
+          "address": eventItem.event.srcAddress,
```

Comment:
Cool 👍🏼, it would also be nice if the user has included txHash in the field selection to log that. Since it's often the easiest thing to track down in a block explorer

Reply:
There's actually an issue for this https://github.com/enviodev/hyperindex/issues/647 😅

But we don't really have it, unless user specifies it in field_selection. Should we maybe add it when we have the value, and omit when we don't? @JonoPrest